### ISSUE TITLE
nway-write-header.py error

### ISSUE THREAD

**Issue by RugiadaC** [JohannesBuchner/nway#5]
I am running Python 2.7.13 and Astropy 2.0.1

When running nway-write-header.py I get the error:

```
Traceback (most recent call last):
  File "../nway-write-header.py", line 7, in <module>
    f[1].header.update('SKYAREA', float(sys.argv[3]))
  File "/Applications/Anaconda/anaconda/lib/python2.7/site-packages/astropy/io/fits/header.py", line 1115, in update
    'a keyword, value, and comment string.'.format(idx))
ValueError: Header update sequence item #0 is invalid; the item must either be a 2-tuple containing a keyword and value, or a 3-tuple containing a keyword, value, and comment string.
```

Changing line 7 to this solved the problem:
`f[1].header.update(SKYAREA=float(sys.argv[3]))`

**Comment by JohannesBuchner** [comment]
I think we could even do `f[1].header['SKYAREA'] = float(sys.argv[3])`

**Comment by JohannesBuchner** [comment]
Fixed in version 3.7 with commit 216457471a46d64bb9109c4a3b98afbf70cb7b50
